src/view/com/composer/GifAltText.tsx

```diff
@@ -200,7 +200,9 @@ function AltTextInner({
           </Text>
           <View style={[a.align_center]}>
             <GifEmbed
-              link={link}
+              thumb={link.thumb}
+              altText={altText}
+              isPreferredAltText={true}
```

Comment:
Note these can just be `altText` and `true` now because we're always passing down the "preferred" version. (In any case they don't really matter because it's a preview and has `hideAlt`. Arguably maybe it shouldn't get any alt at all.)